docs/sharing/embed.md

```diff
@@ -75,11 +75,11 @@ https://5ccbc373887ca40020446347-wtuhidckxo.chromatic.com/iframe.html?id=/story/
 
 <br/>
 
-<iframe src="https://5ccbc373887ca40020446347-wtuhidckxo.chromatic.com/iframe.html?id=shadowboxcta--default&viewMode=docs&shortcuts=false&singleStory=true" width="800" height="400"></iframe>
+<iframe src="https://5ccbc373887ca40020446347-wtuhidckxo.chromatic.com/iframe.html?id=shadowboxcta--docs&viewMode=docs&shortcuts=false&singleStory=true" width="800" height="400"></iframe>
 
 ## Embed stories on other platforms
 
-Every platform has different levels of embed support. Check the documentation of your service to see how they recommend embedding external content.
+Every platform has different levels of embedded support. Check the documentation of your service to see how they recommend embedding external content.
```

Comment:
@jonniebigodes this is intentionally "embed" not "embedded".

Reply:
Already fixed.